modules/book.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
```

Comment:
- [ ] Please, I suggest to avoid disabling the Eslint. ESLint can make your code style consistent, avoid errors, and improve code quality. This means that disabling Eslint will lower your project performance 😞.